---

# Set failed
- set_fact:
    cacheable: true
    sas_local_unix_user_conflicts:
      unreachable: false
      failed: true
      changed: false
  when: local_unix_user_conflicts_reports_generate or local_unix_user_conflicts_facts_generate

# Propogate fail
- fail:
    msg: "{{ result.msg | default('Unexpected error') }}"
